opennebula/helpers.go

```diff
@@ -78,3 +79,16 @@ func isEmptyValue(v reflect.Value) bool {
 	}
 	return false
 }
+
+// NoExists indicate if an entity exists in checking the error code returned from an Info call
+func NoExists(err error) bool {
+
+	respErr, ok := err.(*errors.ResponseError)
+
+	// expected case, the image does not exists so we doesn't return an error
+	if ok && respErr.Code == errors.OneNoExistsError {
```

Comment:
where is the code of this function ?

Reply:
PR comment updated to give more info on the origin of the error code `OneNoExistsError` in goca:
introduced by the commit `4b882f1c073daaf288ae2f26396c21af0df17813` (Jul 11, 2019 during the 5.8 One lifecycle. Thus it should be available in 5.10)